hack/bin/cabal-run-integration.sh

```diff
@@ -61,6 +61,7 @@ run_all_integration_tests() {
       run_integration_tests "$package"
     fi
   done
+  run_integration_tests "stern"
```

Comment:
above the path is hardcoded to `./services/$package`, but stern is underneath "./tools"